packages/fluentui/e2e/e2eApi.ts

```diff
@@ -137,7 +138,10 @@ export class E2EApi {
     modifier?: 'Control' | 'Shift',
   ): Promise<void> => {
     await this.getElement(selector);
+    await this.pressKey(key, modifier);
+  };
 
+  public pressKey = async (key: E2EKeys, modifier?: 'Control' | 'Shift'): Promise<void> => {
```

Comment:
Let's avoid restoration of `pressKey` as it was replaced with `waitForSelectorAndPressKey` intentionally